landoscript/tests/test_android_l10n_import.py

```diff
@@ -255,6 +240,85 @@ async def test_success(
+                    "object": {
+                        "entries": [
+                            {
+                                "name": "components",
+                                "type": "tree",
+                                "object": {
+                                    "entries": [
+                                        {
+                                            "name": "browser",
+                                            "type": "tree",
+                                            "object": {
+                                                "entries": [
+                                                    {
+                                                        "name": "toolbar",
+                                                        "type": "tree",
+                                                        "object": {
+                                                            "entries": [
+                                                                {
+                                                                    "name": "src",
+                                                                    "type": "tree",
+                                                                }
+                                                            ],
+                                                        },
+                                                    }
+                                                ],
+                                            },
+                                        }
+                                    ],
+                                },
+                            }
+                        ],
+                    }
+                },
+            },
+        },
+        {
+            "data": {
+                "repository": {
+                    "object": {
+                        "entries": [
+                            {
+                                "name": "main",
+                                "type": "tree",
+                                "object": {
+                                    "entries": [
+                                        {
+                                            "name": "res",
+                                            "type": "tree",
+                                            "object": {
+                                                "entries": [
+                                                    {
+                                                        "name": "values",
+                                                        "type": "tree",
+                                                        "object": {
+                                                            "entries": [
+                                                                {
+                                                                    "name": "strings.xml",
+                                                                    "type": "blob",
```

Comment:
![image](https://github.com/user-attachments/assets/9e6368b6-5113-4d56-b102-4bc7d2440d0c)


sorry, i had to :)